src/launcher/reducers/__tests__/autoUpdateReducer-test.js

```diff
@@ -19,6 +19,18 @@ import * as AutoUpdateActions from '../../actions/autoUpdateActions';
 import reducer from '../autoUpdateReducer';
 
 const initialState = reducer(undefined, {});
+const { CancellationToken } = require('electron-updater');
+
+describe('cancellactionToken', () => {
```

Comment:
Typo in the test description, but it's quite benign so I think it's ok ;)

Reply:
What, nice catch! Will fix ^^